plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -210,21 +210,24 @@ class ServicesTable extends React.Component {
     ];
 
     return (
-      <Dropdown
-        key="actions-dropdown"
-        anchorRight={true}
-        buttonClassName="button button-mini dropdown-toggle button-link table-display-on-row-hover"
-        dropdownMenuClassName="dropdown-menu"
-        dropdownMenuListClassName="dropdown-menu-list"
-        dropdownMenuListItemClassName="clickable"
-        wrapperClassName="dropdown flush-bottom table-cell-icon"
-        items={dropdownItems}
-        persistentID={ServiceActionItem.MORE}
-        onItemSelection={this.onActionsItemSelection.bind(this, service)}
-        scrollContainer=".gm-scroll-view"
-        scrollContainerParentSelector=".gm-prevented"
-        transition={true}
-        transitionName="dropdown-menu" />
+      <Tooltip interactive={true} content={'More actions'}>
+        <Dropdown
+          key="actions-dropdown"
```

Comment:
Let's kill the `key` property. It was needed previously because this element was an element in an array, whereas now it's being rendered directly in JSX.